.travis.yml

```diff
@@ -12,13 +19,20 @@ env:
 before_install:
   - rvm install 2.3.1
   - if [[ `npm -v` != 3* ]]; then npm i -g npm@3; fi
-  - npm install -g bower grunt-cli
+  - npm install -g bower
```

Comment:
Unsure - we can take a look at that as well.   Can we do that after this get in?